### ISSUE TITLE
Ubuntu: no space left on device

### ISSUE THREAD

**Issue by rjoleary** [u-root/webboot#247]
**Describe the bug**
The ISO download is interrupted with the error "write /tmp/iso-download-758024287: no space left on device".

**To Reproduce**
On an x230 laptop, select the Ubuntu distro and wait while it downloads. The laptop has 4GB of RAM.

**Comment by rjoleary** [comment]
My USB driver is one large partition:

```
Device     Boot Start       End   Sectors  Size Id Type
/dev/sda1  *     2048 121061375 121059328 57.7G ef EFI (FAT-12/16/32)
```

This is more than enough to hold the image. I have a feeling we are downloading to RAM instead of the USB stick.

**Comment by kellysun878** [comment]
Fixed by #272.
